src/send_kcidb.py

```diff
@@ -486,12 +488,61 @@ def _get_test_data_recursively(self, node, origin, parsed_test_node, parsed_buil
                 self._get_test_data_recursively(child, origin, parsed_test_node,
                                                 parsed_build_node)
 
+    def _node_processed(self, node):
+        """
+        Mark the node as processed, sent_kcidb field to True
+        That means kcidb has received the node, and at least tried to process it
+        Data in node might be invalid, and not really sent to kcidb
+        This is workaround, until we improve event handling in kernelci-pipeline/api
+        """
+        node['processed_by_kcidb_bridge'] = True
```

Comment:
I though we were going to have 2 flags, i.e. one is for successful submission and another one is for processing.

Reply:
for submission not yet, as it will be next step of handling errors, which is unclear if happening at all